manager/assets/modext/widgets/resource/modx.panel.resource.js

```diff
@@ -541,7 +541,10 @@ Ext.extend(MODx.panel.Resource,MODx.FormPanel,{
             ,name: 'template'
             ,id: 'modx-resource-template'
             ,anchor: '100%'
-            ,editable: false
+            ,editable: true
+            ,typeAhead: true
```

Comment:
It fills the first available template (on your case: BaseTemplate) if you type non-existed template.
If you have "**Basic**" template and you start to type "**b**", it will select the first "**b**" template when you stop typing for 250ms ([default](http://docs.sencha.com/extjs/3.4.0/#!/api/Ext.form.ComboBox-cfg-typeAheadDelay)), but you **can** keep continue typing, the blue highlight is just **suggestion**
![typeahead template](https://cloud.githubusercontent.com/assets/308524/12616085/dc4edf9e-c53b-11e5-8759-4e1fb592425b.png)